types/2020-03-02/SubscriptionSchedules.d.ts

```diff
@@ -253,10 +253,15 @@ declare module 'stripe' {
            */
           plan: string | Stripe.Plan | Stripe.DeletedPlan;
 
+          /**
+           * ID of the price to which the customer should be subscribed.
+           */
+          price?: string | Stripe.Price | Stripe.DeletedPrice;
```

Comment:
Yes this is expected. It wasn't in node because we merged before we finished the change we put in Java here https://github.com/stripe/stripe-java/blob/master/src/main/java/com/stripe/model/SubscriptionSchedule.java#L730-L734